### DISCUSSION TITLE
What's the best way to access colors programmatically?

### DISCUSSION THREAD

**Discussion by toniengelhardt** [unocss/unocss#1966] [Q&A] [answered]
Sometimes we need to access the colors in code, e.g. for chart.js. What's the best way to do that?

**Comment by toniengelhardt** [comment, marked as answer]
I found a partial answer here: https://github.com/unocss/unocss/discussions/714.

**Comment by ManUtopiK** [comment]
Maybe this response fit best : https://github.com/unocss/unocss/discussions/1908#discussioncomment-4217986
